src/Uno.UI/UI/Xaml/Controls/Flyout/Flyout.cs

```diff
@@ -20,12 +20,10 @@ namespace Windows.UI.Xaml.Controls
 {
 	[ContentProperty(Name = "Content")]
 	public partial class Flyout : FlyoutBase
```

Comment:
I didn't realise you were modifying the Flyout class aswell, are those changes really necessary?